src/AdminNotice.php

```diff
@@ -238,26 +243,80 @@ public function urgency($urgency): self
         return $this;
     }
 
+    /**
+     * Alias for setting the urgency to info
+     *
+     * @since 1.1.0
+     */
     public function asInfo(): self
     {
         return $this->urgency(NoticeUrgency::info());
     }
 
+    /**
+     * Alias for setting the urgency to success
+     *
+     * @since 1.1.0
+     */
     public function asSuccess(): self
     {
         return $this->urgency(NoticeUrgency::success());
     }
 
+    /**
+     * Alias for setting the urgency to warning
+     *
+     * @since 1.1.0
+     */
     public function asWarning(): self
     {
         return $this->urgency(NoticeUrgency::warning());
     }
 
+    /**
+     * Alias for setting the urgency to error
+     *
+     * @since 1.1.0
+     */
     public function asError(): self
     {
         return $this->urgency(NoticeUrgency::error());
     }
 
+    /**
+     * Uses the alternate WP notice styles
+     *
+     * @unreleased
+     */
+    public function alternateStyles(bool $altStyle = true): self
+    {
+        $this->alternateStyles = $altStyle;
+
+        return $this;
+    }
+
+    /**
+     * Uses the standard WP notice styles
+     *
+     * @unreleased
+     */
+    public function standardStyles(): self
+    {
+        $this->alternateStyles = false;
+
+        return $this;
+    }
+
+    /**
+     * Returns whether the notice uses the alternate WP notice styles
+     *
+     * @unreleased
```

Comment:
```suggestion
     * @unreleased
```
Do these need to be updated?